Example/Example/EditorDemoController.swift

```diff
@@ -1032,9 +1032,48 @@ extension EditorDemoController {
 
 extension EditorDemoController: TextViewAttachmentDelegate {
 
+    func exportPreviewImageForVideo(atURL url: URL, onCompletion: @escaping (UIImage) -> (), onError: @escaping () -> ()) {
```

Comment:
This method is not part of `TextViewAttachmentDelegate`.  Can we move it to a more appropiate location?